src/pyird/plot/showspec.py

```diff
@@ -0,0 +1,18 @@
+import matplotlib.pyplot as plt
+
+def show_wspec(df_wspec,**kwargs):
```

Comment:
Maybe we should be a little more careful with naming. I have not paid much attention to this in the past, but, I change my mind after reading [this article](https://kidger.site/thoughts/jax-vs-julia/).

> Academic code quality is famously poor, and the Julia ecosystem is no exception. Here’s a small sample of the issues plaguing the ecosystem:

> Many Julia APIs look like Optimiser(η=...) rather than Optimiser(learning_rate=...). This is a pretty unreadable convention.

Then, I bought [this book](https://www.amazon.co.jp/%E3%83%AA%E3%83%BC%E3%83%80%E3%83%96%E3%83%AB%E3%82%B3%E3%83%BC%E3%83%89-%E2%80%95%E3%82%88%E3%82%8A%E8%89%AF%E3%81%84%E3%82%B3%E3%83%BC%E3%83%89%E3%82%92%E6%9B%B8%E3%81%8F%E3%81%9F%E3%82%81%E3%81%AE%E3%82%B7%E3%83%B3%E3%83%97%E3%83%AB%E3%81%A7%E5%AE%9F%E8%B7%B5%E7%9A%84%E3%81%AA%E3%83%86%E3%82%AF%E3%83%8B%E3%83%83%E3%82%AF-Theory-practice-Boswell/dp/4873115655/ref=sr_1_1?adgrpid=52747835709&gclid=CjwKCAjw7cGUBhA9EiwArBAvokI9YHjeMPGRtLv-EewHFwMEI8_2i_W8uv7WxB67uEJVsdi34MdVLBoC92gQAvD_BwE&hvadid=553962830540&hvdev=c&hvlocphy=1009317&hvnetw=g&hvqmt=e&hvrand=10865635296344883096&hvtargid=kwd-334307148361&hydadcr=16038_13486739&jp-ad-ap=0&keywords=%E3%83%AA%E3%83%BC%E3%83%80%E3%83%96%E3%83%AB%E3%82%B3%E3%83%BC%E3%83%89&qid=1653689417&sr=8-1).

So back to this name, for instance, `plot_wavcal_spectrum` is better? What do you think of? Anyway, we need to discuss the naming/refactoring in pyird because there are many unclear names (partly inherited from IRAF though...)

Reply:
I agree with your opinion. I have been naming them without thinking it through, so I will think about it.

Reply:
Do you wanna rename in this PR? If so, I will merge the PR after the renaming. @YuiKasagi

Reply:
I have just renamed it. Thank you for your suggestion!